server/lib/topic.js

```diff
@@ -61,13 +63,15 @@ const channels = {
 // writes events into the event database table
 function databaseLogger(data) {
 
-  let record = {
+  const record = {
     timestamp: new Date(data.timestamp),
     user_id: data.user_id,
     channel: data.channel,
+    entity : data.entity.toUpperCase(),
```

Comment:
Can you think of an instance when it shouldn't be required?  So far, it has been used in each event published.